fluent.ebnf

```diff
@@ -32,15 +32,17 @@ quoted-text          ::= ([^{"] | '\{' | '\"')+
 block-text           ::= NL __ '|' unquoted-pattern
 
 placeable            ::= '{' __ expression __ '}'
-expression           ::= quoted-pattern
+expression           ::= selector-expression | select-expression
+
+selector-expression  ::= quoted-pattern
                        | number
                        | identifier
                        | variable
-                       | select-expression
                        | member-expression
                        | call-expression
+                       | placeable
```

Comment:
The use case is tightly related to #4. In particular, we need a way to remove the ambiguity from `call-expressions` with multiple arguments, some of them being `select-expressions`:

```
key = { LIST(foo, { bar, baz -> 
        [bar, baz] Bar and Baz
    }) }
```